src/components/Form/index.tsx

```diff
@@ -36,7 +36,7 @@ const Form = ({ title }: Form) => {
         <input type="text" className="form__input" />
       </div>
       <div className="form__actionable">
-        <Link href="/registration-success">
+        <Link href="/sign-up-success">
```

Comment:
There will surely be other logics rather than just navigation, I would consider using Next Router instead.